diesel_cli/README.md

```diff
@@ -7,17 +7,20 @@ bi-directional changes to your database that get applied sequentially.
 Installation
 ---------------
 
-The diesel cli, by default, requires [`libpq`][postgres], [`sqlite`][sqlite]
+The diesel cli, by default, requires [`openssl`][openssl] [`libpq`][postgres], [`sqlite`][sqlite]
```

Comment:
You should add a comma after `openssl`. And for readability, it may be good that the lines aren't long like this.

```
The diesel cli, by default, requires [`openssl`][openssl], [`libpq`][postgres],
[`sqlite`][sqlite] and [`mysql`][mysql]. Once these dependencies are installed,
you can run `cargo install diesel_cli`.
```

Reply:
Adding comma, but not sure how I would break it up? Also, when rendered, it's significantly shorter.

Reply:
In the editor, long lines aren't good to read. At a quick glance, many other lines are separated to no more than 80 characters. You can shorten the length of a line by separating newlines instead of a space where appropriate.

Reply:
Fixed, I forgot Markdown doesn't respect a single newline, which I can now appreciate!